.github/workflows/django.yml

```
@@ -0,0 +1,30 @@
+name: Django CI
+
+on:
```

YAMLlint (1.38.0)

[warning] truthy value should be one of [false, true]

(truthy)